revolution_pi/gpio_pin.go

```diff
@@ -145,56 +233,61 @@ func (pin *gpioPin) PWMFreq(ctx context.Context, extra map[string]interface{}) (
 	if !pin.initialized {
 		return 0, errors.New("pin not initialized")
 	}
+	if !pin.isOutputPWM() && !pin.isOutputWord() {
+		return 0, fmt.Errorf("cannot get PWM Frequency, Pin %s is not a PWM pin", pin.Name)
+	}
```

Comment:
This is _almost_ copied and pasted from before. Is it also supposed to check if `pin.pwmMode` like the others? If so, we definitely should put it in a helper function so we don't have bugs by omission like this.

Reply:
makes sense to switch this to check pwmMode, but I don't quite understand why we would want a helper for a boolean

Reply:
Why do you need to check if the pin is not `isOutputWord` ?

Reply:
because the way we setup the module OutputWord and PWM addresses are aliased to eachother, so if the pin is not one of those two options then I know I can reject the pwm request